core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/multitenancy/utils/TenantAxisUtils.java

```diff
@@ -455,8 +455,16 @@ public static void cleanupTenants(long tenantIdleTimeMillis) {
                                 PrivilegedCarbonContext carbonContext = PrivilegedCarbonContext.getThreadLocalCarbonContext();
                                 carbonContext.setTenantDomain(tenantDomain, true);
 
-                                // Terminating idle tenant configuration contexts.
-                                terminateTenantConfigContext(tenantCfgCtx);
+                                if (carbonContext.getTenantId() != MultitenantConstants.INVALID_TENANT_ID) {
+                                    // Terminating idle tenant configuration contexts.
+                                    terminateTenantConfigContext(tenantCfgCtx);
+                                } else {
+                                    if (log.isDebugEnabled()) {
+                                        log.debug("Tenant ID of tenant domain " + tenantDomain +
```

Comment:
Seems like this log can be a single line. If single line will not exceed 120 characters let's keep that in a single line.

Reply:
log exceed 120 characters and can not keep that in a single line.